documentation/modules/ROOT/pages/configuration/storage.adoc

```diff
@@ -0,0 +1,721 @@
+
+|[[retry-max-attempts]]<<retry-max-attempts, `offset.storage.jdbc.connection.retry.max.attempts`>>
+|5
+|(Optional) Specifies the maximum number of times that {prodname} retries the connection to the offset storage database after a connection failure.
+
+|[[jdbc-offset-table-name]]<<jdbc-offset-table-name, `offset.storage.jdbc.table.name`>>
+|`debezium_offset_storage`
+|The name of the table where {prodname} stores offsets.
+
+|[[jdbc-offset-table-ddl]]<<jdbc-offset-table-ddl, `offset.storage.jdbc.table.ddl`>>
+|xref:jdbc-offset-table-ddl-default[(Table Structure)]
+|DDL statement to create the offset table.
+
+|[[jdbc-offset-table-select]]<<jdbc-offset-table-select, `offset.storage.jdbc.table.select`>>
+|xref:jdbc-offset-table-ddl-default[(Select query)]
+|DML statement that {prodname} uses to read offsets values from the table.
+
+|[[jdbc-offset-table-insert]]<<jdbc-offset-table-insert, `offset.storage.jdbc.table.insert`>>
+|xref:jdbc-offset-table-insert-default[(Insert query]
+|DML statement that {prodname} uses to write offsets to the table.
+
+|[[jdbc-offset-table-delete]]<<jdbc-offset-table-delete, `offset.storage.jdbc.table.delete`>>
+|xref:jdbc-offset-table-delete-default[(Delete query)]
+|DML statement that {prodname} uses to remove offsets from the table.
+
+|===
+
+==== Deprecated configuration prior to 3.2
+[cols="35%a,10%a,55%a",options="header"]
+|===
+|Property
+|Default
+|Description
+
+|[[deprecated-jdbc-offset-type]]<<deprecated-jdbc-offset-type, `offset.storage`>>
+|No default
+|Must be set to `io.debezium.storage.jdbc.offset.JdbcOffsetBackingStore`.
+
+|[[deprecated-jdbc-offset-url]]<<deprecated-jdbc-offset-url, `offset.storage.jdbc.url`>>
+|No default
+|JDBC driver connection string to connect to the database.
+
+|[[deprecated-jdbc-offset-user]]<<deprecated-jdbc-offset-user, `offset.storage.jdbc.user`>>
+|No default
+|(Optional) The username through which {prodname} connects to the database that stores offset data.
+
+|[[deprecated-jdbc-offset-password]]<<deprecated-jdbc-offset-password, `offset.storage.jdbc.password`>>
+|No default
+|(Optional) Password for the user specified by xref:deprecated-jdbc-offset-user[`offset.storage.jdbc.user`].
+
+|[[deprecated-wait-retry-delay-ms]]<<deprecated-wait-retry-delay-ms, `offset.storage.jdbc.wait.retry.delay.ms`>>
+|3 seconds
+|(Optional) Specifies the time, in milliseconds, that the connector waits to retry the connection after failed attempts to connect to the offset storage database.
+
+|[[deprecated-retry-max-attempts]]<<deprecated-retry-max-attempts, `offset.storage.jdbc.retry.max.attempts`>>
+|5
+|(Optional) Specifies the maximum number of times that {prodname} retries the connection to the offset storage database after a connection failure.
+
+|[[deprecated-jdbc-offset-table-name]]<<deprecated-jdbc-offset-table-name, `offset.storage.jdbc.offset.table.name`>>
+|`debezium_offset_storage`
+|The name of the table where {prodname} stores offsets.
+
```

Comment:
I modified the targets for the remaining links in this table to match the values in the link text. Check that I've targeted the correct property names.